sandbox/build_vrt.py

```diff
@@ -0,0 +1,40 @@
+import os
+from time import time
+
+from osgeo import gdal
+
+tic = time()
+
+WORKING_DIRECTORY = 'output/det/vaud/tiles_z18/'
+os.chdir(WORKING_DIRECTORY)
+
+print('Create VRT file for each year')
+for year in range(1980, 1981):
+    SOURCE_FOLDER = str(year)
```

Comment:
SOURCE_FOLDER = str(year)+'/all-images'

Reply:
Yes, it is better to change working directory. I think my comment was because I thought that images to mosaic in this project are more likely to be in a `YEAR/all-images/` folder, than simply in a `YEAR` folder.

Since the script is meant to stay in the sandbox, this is not critical.